scripts/sysman/firmware.yml

```diff
@@ -31,6 +31,31 @@ members:
       name: "version[$S_STRING_PROPERTY_SIZE]"
       desc: "[out] NULL terminated string value. The string \"unknown\" will be returned if this property cannot be determined."
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Intel Firmware Image (IFWI) version"
```

Comment:
could you make it experimental?

Reply:
@jandres742 moved to experimental extension